app/data/repositories/DecksRepository.py

```diff
@@ -18,8 +21,11 @@ def create(user_id: int, name: str) -> Optional[Deck]:
     deck_id = c.lastrowid
 
     if deck_id:
+        logger.debug(
+            "Deck with id:{deck_id} has been saved into database successfully ")
         return Deck(deck_id, user_id, name)
 
+    logger.error("Saving deck into database failed")
```

Comment:
there is no exception? are you able to log error from db?

Reply:
like above I think that I am able

Reply:
do it or todo

Reply:
TODO, done